utils/scripts/dbaas_utils.py

```diff
@@ -372,3 +372,35 @@ def create_provision_instance_yaml(isv, prov_acc_ns, deploy_instance_ns):
         data["spec"]["otherInstanceParams"]["projectName"] = instance
     print(data)
     return yaml.dump(data, sort_keys=False)
+
+
+def add_policy_connection_namespaces(namespace: list):
+    """Adding Connection Namespace on DBaaSPolicy"""
+    sl = BuiltIn().get_library_instance("SeleniumLibrary")
+    btn_add_conn_xpath = "//button[@id='root_spec_connectionNamespaces_add-btn']"
+    if not namespace:
+        namespace = "*"
```

Comment:
Can you please explain what '*' means here? Does it mean that this ConnectionNS can be seen by all other namespaces?

Reply:
Yes, only if it is not specified and this method is being invoked.